vtr_flow/arch/titan/README.rst

```diff
@@ -24,46 +26,72 @@ CAD evaluation).
 
 Directory Structure
 --------------------------------------------------
+
 stratixiv_arch.timing.xml:
-    A capture of Altera's Stratix IV FPGA architecture. It makes some 
-    relatively minor architectural approximations to be compatible with
-    VPR.  It includes a timing model which has been calibrated to the
-    Stratix IV timing model included in Altera's Quartus II CAD tools.
+    An enhanced capture of Altera's Stratix IV FPGA architecture. It makes some 
+    relatively minor architectural approximations to be compatible with VPR. It
+    includes a timing model which has been calibrated to the Stratix IV timing
+    model included in Altera's Quartus II CAD tools.
+
+    Use this architecture file, unless you have specific reasons to use the
+    other ones in this directory.
+
+stratixiv_arch.timing.complex_sb.12to1.xml:
+    A variant of the above architecture capture using customized switch-block
+    with hierarchical wire connectivity and fixed 12:1 L4 and L16 driver muxes.
+
+stratixiv_arch.timing.complex_sb.L16_40to1.L4_turn-straight_rand_L4_L16.xml:
+    A variant like stratixiv_arch.timing.complex_sb.12to1.xml, but increased the
+    L16 driver muxes to 40:1.
+
+stratixiv_arch.timing.complex_sb.L16_72to1.L4_turn-straight_rand_L4_L16.xml:
+    A variant like stratixiv_arch.timing.complex_sb.12to1.xml, but increased the
+    L16 driver muxes to 72:1.
+
+stratixiv_arch.timing.complex_sb.L4_16to1.L16_72to1.L4_turn-straight_rand_L4_L16.xml:
+    A variant like stratixiv_arch.timing.complex_sb.12to1.xml, but increased the
+    L4 driver muxes to 16:1 and the L16 driver muxes to 72:1.
+
+For more details about the Altera's Stratix IV FPGA architecture capture variants,
+see Section 5.4 of the paper, "VTR 8: High Performance CAD and Customizable FPGA Architecture Modelling": 
+<https://www.eecg.utoronto.ca/~kmurray/vtr/vtr8_trets.pdf>
+
+**legacy subdirectory:**
 
-    Use this architecture file, unless you have specific reasons to use 
-    stratixiv_arch.detailed.xml or stratixiv_arch.simple.xml.
+    stratixiv_arch.timing.legacy.xml:
+        The old capture of Altera's Stratix IV FPGA architecture.
```

Comment:
This architecture was used to generate the results in the Timing-driven Titan paper: 
K. Murray et al, "Timing-Driven Titan: Enabling Large Benchmarks and Exploring the Gap between Academic and Commercial CAD," ACM TRETS, March 2015, https://dl.acm.org/doi/10.1145/2629579

Reply:
Also added these comments as suggested.